contrib/vagrant/scripts/helpers.bash

```diff
@@ -72,7 +72,7 @@ cluster_dns_ipv6=${K8S_CLUSTER_DNS_IPV6:-"FD03::A"}
 cluster_api_server_ipv4=${K8S_CLUSTER_API_SERVER_IPV4:-"172.20.0.1"}
 cluster_api_server_ipv6=${K8S_CLUSTER_API_SERVER_IPV6:-"FD03::1"}
 
-k8s_version="v1.21.0"
+k8s_version="v1.22.0-rc.0"
```

Comment:
Do we test if the upgraded k8s versions don't break vagrant workflows like the dev VM?